jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/Response.java

```diff
@@ -150,21 +150,15 @@ interface ContentListener extends AsyncContentListener
          *
```

Comment:
Please add that upon returning normally from the method, there is automatic demand.

Reply:
Done.